olp-cpp-sdk-core/include/olp/core/utils/LruCache.h

```diff
@@ -286,50 +390,52 @@ class LruCache {
   }
 
   /**
-   * @brief find an value in the cache
+   * @brief Finds a value in the cache.
+   *
+   * @note This function does NOT promote the item pointed to by a key if found.
    *
-   * Note - this function will NOT promote the item pointed to by key if found
+   * @param key The key to find.
    *
-   * @param key the key to find
-   * @return an iterator to the value if found, otherwise, an iterator pointing
-   * to end()
+   * @return If found, the iterator to the value; the iterator pointing
+   * to `end()` otherwise.
    */
   const_iterator FindNoPromote(const Key& key) const {
     auto it = map_.find(key);
     return const_iterator{it};
   }
 
   /**
-   * @brief find an value in the cache
+   * @brief Finds a value in the cache.
    *
-   * Note - this function will promote the item pointed to by key if found
+   * @note This function promotes the item pointed to by a key if found.
    *
-   * @param key the key to find
-   * @param nullValue the value to return if the key/value pair is not in the
+   * @param key The key to find.
+   * @param nullValue The value to return if the key-value pair is not in the
    * cache
-   * @return a const reference to the value, or nullValue if not found
+   * @return If found, a constant reference to the value; `nullValue` otherwise.
    */
   const Value& Find(const Key& key, const Value& nullValue) {
     auto it = Find(key);
     return it == end() ? nullValue : it.value();
   }
 
-  /// returns the begin iterator.
+  /// Returns the `begin` iterator.
   const_iterator begin() const { return const_iterator{first_}; }
 
-  /// returns the end iterator
+  /// Returns the `end` iterator.
   const_iterator end() const { return const_iterator{map_.end()}; }
 
-  /// reverse begin iterator
+  /// Reverses the `begin` iterator.
   const_iterator rbegin() const { return const_iterator{last_}; }
 
-  /// reverse end iterator
+  /// Reverses the `end` iterator.
   const_iterator rend() const { return const_iterator{map_.end()}; }
```

Comment:
Should be rather:
- Returns a [reverse] constant iterator to the beginning.
- Returns a [reverse] constant iterator to the end.

Reply:
Changed.